coursedata/level-defaults/es.yaml

```diff
@@ -51,13 +51,13 @@
             nombre is Hedy
             print Bienvenido nombre
     -   name: "ask"
-        explanation: "Preguntar algo con `ask`."
-        example: "Ejemplo: color is `ask` ¿Cuál es tu color favorito?"
+        explanation: "Preguntar algo con `ask`. ¡Atención! Necesitas darle un nombre a la respuesta con `is`"
+        example: "Ejemplo: color is ask ¿Cuál es tu color favorito?"
         demo_code: |-
             color is ask ¿Cuál es tu color favorito?
-            print color es tú favorito
+            print El color es tú color favorito
```

Comment:
Here I would leave the original line, because both instances of `color` will be replaced by the variable value (so this would read, for example, "el rojo es tu rojo favorito"

Reply:
I see, didn't think about that 🤔..

Reading "rojo es tu favorito" sounds incomplete to me.
Maybe we could rename the variable color so we can have the complete sentence?
For example:
El `nombre_color` es tu color favorito.

I know the variable name has to be as simple as possible, but if it clashes with an actual world you want to use, maybe it's not a bad idea to rename it to `nombre_color` or something similar?

Reply:
Not a bad idea at all! Although I think @Felienne did that on purpose to create "tension" of sorts, that can be resolved in a later level where things can be quoted.

@Felienne should we aim to have a complete sentence here saying "Color X is your favorite color" or do we want to leave the tension of not being able to say "color" if we use it as a variable name?

Reply:
@fpereiro After working on section 3, I see that was @Felienne's intention indeed.

So I'd leave it as `El color es tu favorito`